style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: red;
+}
```

Comment:
- [ ] As requested by the previous reviewer, the CSS file is supposed to include a CSS class selector styling the h1 element. This involves having the h1 with a class attribute and styling it using the class selector and not the element selector. For more information on how to do that please visit [using a class selector to style elements](w3schools.com/cssref/sel_class.php)